libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -457,6 +457,18 @@ bool AP_Arming::gps_checks(bool report)
     const AP_GPS &gps = AP::gps();
     if ((checks_to_perform & ARMING_CHECK_ALL) || (checks_to_perform & ARMING_CHECK_GPS)) {
 
+        // Any failure messages from GPS backends
+        if ((checks_to_perform & ARMING_CHECK_ALL) ||
+            (checks_to_perform & ARMING_CHECK_GPS_CONFIG)) {
```

Comment:
use ARMING_CHECK_GPS not _CONFIG I think, it is about finding right GPS modules, not configuring them